app/build.gradle.kts

```diff
@@ -2,6 +2,7 @@ plugins {
   id("com.android.application")
   id("org.jetbrains.kotlin.android")
   id("io.github.jackchen365.sample")
+  id("org.jlleitschuh.gradle.ktlint")
```

Comment:
As I mentioned, We better add this `org.jlleitschuh.gradle.ktlint-idea` as well.

Therefore, We can have those two tasks.

<img width="345" alt="image" src="https://user-images.githubusercontent.com/12761044/182051652-40cff788-b9e1-4f6c-8f62-4135fdb42241.png">

Reply:
In fact, the ktlint core plugin already apply the plugin:
```kotiln
open class KtlintPlugin : Plugin<Project> {
    override fun apply(target: Project) {
        // Apply the idea plugin
        target.plugins.apply(KtlintIdeaPlugin::class.java)
        ......
    }
    ......
}
```

Reply:
But, if we need to enable add git hook for these tasks, we need to apply it in the root project.